server/utils/api/tts.ts

```diff
@@ -0,0 +1,10 @@
+import type { UserDocData } from './user'
+import type { UserSession } from '#auth-utils'
+
+export function isTTSAvailable(session: UserSession, userDoc: UserDocData | undefined): boolean {
+  if (!session || !session.user) return false
+  const isLikerPlus = session.user.isLikerPlus || false
+  if (isLikerPlus) return true
```

Comment:
I want to simplify the input params and follow h3/nuxt-auth utils
```suggestion
import type { H3Event } from 'h3'

export async function getUserTTSAvailable(event: H3Event): Promise<boolean> {
  const session = await getUserSession(event)
  if (!session || !session.user) return false
  const isLikerPlus = session.user.isLikerPlus || false
  if (isLikerPlus) return true
  const userDoc = await getUserDoc(session.user.evmWallet)
```



Caller
```ts
const isTTSAvailable = await getUserTTSAvailable(event)
```